velox/vector/DecodedVector.cpp

```diff
@@ -349,7 +349,7 @@ void DecodedVector::setBaseData(
     case VectorEncoding::Simple::LAZY:
       break;
     case VectorEncoding::Simple::FLAT: {
-      data_ = vector.values()->as<void>();
+      data_ = vector.values() ? vector.values()->as<void>() : nullptr;
```

Comment:
nit: perhaps, add a comment to explain that all-nulls vector may have have a nullptr values buffer